src/crudop.js

```diff
@@ -10,10 +11,29 @@ if (localStorage.length > 0) {
 }
 
 // update UI
-const updateUI = () => {
+export const updateUI = () => {
+  let dyclass = '';
+  let classdy = '';
+  let cross = '';
   for (let i = 0; i < taskArr.length; i += 1) {
+    if (taskArr[i].completed) {
+      dyclass = '';
+      classdy = 'active';
+      cross = 'cross';
+    } else {
+      dyclass = 'active';
+      classdy = '';
```

Comment:
- [ ] When I select one task to be completed, the other task automatically gets a cross sign when refreshed, the data, and its value remain correct but in UI it got a `cross` sign as I have shown in the below screenshot. Please fix this.

![Screenshot from 2022-07-09 01-13-08](https://user-images.githubusercontent.com/26767383/178062013-f586bb1d-f19e-4a80-bc68-923de0c9542e.png)

Reply:
Hello @Deepakdanger, it is a little bit confusing for me how to address that because this error doesn't appear on my side but I will try my best
Thank you!